### ISSUE TITLE
Configuration Mode for Environment Profiles in the Editor

### ISSUE THREAD

**Issue by thiagosantoscunha** [Microsoft/vscode#72520]
VSCode is an incredible tool that supports different types of developments. This is good for developers in Brazil, where at one point it develops in a stack, and often in parallel, it uses another set of stacks to develop.

VSCode Extensions help us manage this, however, the more stacks of environments, the more time-consuming to manage the workspace, and the slower the text editor gets.

Creating extension packages can be a good solution, but it is a package that downloads a set of extensions. So imagine the following scenario:

- The first moment: I'm developing frontend with a set of extensions for Javascript, HTML and CSS / SASS.

- The second moment: in parallel, I need to develop a project using a set of extensions that help PHP development.

- The third moment: in parallel, I need to develop a project using a set of extensions that help DotNet Core development.

- The third moment: in parallel, I need to develop a project using a set of extensions that help Python development.

- Etc...

Well ... having to manage all these plugins gets a bit complex, because, on average, they would give more than 60 extensions together.

One of the possible solutions would be to create user profiles along with the environment profile to aid in this management process. If I'm working in a frontend environment, I load the profile according to this stack, enabling the required extensions and disabling the unnecessary one.

**Comment by sandy081** [comment]
Duplicate of https://github.com/Microsoft/vscode/issues/57548
